lib/networkCall.js

```diff
@@ -0,0 +1,17 @@
+const fetch = require('node-fetch')
+
+const networkCall = async (url, method = 'get', body = null, headers = null) => {
+  try {
+    const response = await fetch(url, {
+      method: method,
+      body: body,
+      headers: headers
+    })
+    const json = await response.json()
+    return json
+  } catch (err) {
+    return err
```

Comment:
When we are returning error, the error will be obtained in the then block only. We will need to throw it from here.
Also, can we handle throwing of 4XX and 5XX errors too.
Refer the limitation of the library here: https://medium.com/@thejasonfile/fetch-vs-axios-js-for-making-http-requests-2b261cdd3af5

Reply:
Error throwing done.